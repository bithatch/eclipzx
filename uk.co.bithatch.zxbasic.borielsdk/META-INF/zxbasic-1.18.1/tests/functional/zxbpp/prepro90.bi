
#define MACRO YES

#if MACRO != YES
   WRONG
#else
   OK
#endif
