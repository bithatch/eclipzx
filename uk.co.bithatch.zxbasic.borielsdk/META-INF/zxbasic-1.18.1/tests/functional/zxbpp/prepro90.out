#line 1 "prepro90.bi"

#line 3 "prepro90.bi"

#line 7 "prepro90.bi"
   OK
#line 9 "prepro90.bi"
